apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.cs

```diff
@@ -24,6 +24,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Xunit;
+using static Google.Cloud.Spanner.V1.SessionPool;

```

Comment:
I'm not sure why this is needed. If it is, that's fine - but worth checking.

Reply:
@jskeet  Its required because we are creating a SessionPoolSegementKey() on line 236. If we omit this we need to create it like SessionPool.SessionPoolSegmentKey().

Reply:
Ah right. Yes, that makes sense. (I'm still unclear why the on in integration tests is needed though.)